webapp/certification/views.py

```diff
@@ -176,6 +176,7 @@ def certification_home():
         limit = request.args.get("limit", default=20, type=int)
         offset = request.args.get("offset", default=0, type=int)
         filters = request.args.get("filters", default=False, type=bool)
+        vendor_page = request.args.get("vendor_page", default=False, type=bool)
```

Comment:
The problem I see with this, is that the user has no way to get out of the vendor page? Maybe something in the UI could help removing that `vendor_page` or resetting the URL

Reply:
Yeah I see what you mean. In the wireframe the use case is only for vendors to be able to link to this page from their sites, it doesn't mention how to get to this page from the search results page (hence us adding the vendor column in the table). I think we will need UX to help